rollup.config.js

```diff
@@ -1,16 +1,39 @@
-import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
 import json from 'rollup-plugin-json';
+import replace from 'rollup-plugin-replace';
+import resolve from 'rollup-plugin-node-resolve';
+
+const version = 
+  process.env.rpc ? 'rpc' : 
+  process.env.lib ? 'lib' : 
+  process.env.cli ? 'cli' : 'invalid';
+
+if (version === 'invalid') {
+  throw new Error('Provide a valid output version via --environment');
+}
 
 export default {
-  entry: 'prettier-rpc.js',
-  dest: 'dist/prettier-rpc-' + require('./package.json').dependencies.prettier + '.min.js',
+  entry: `prettier-${version}.js`,
+  dest: `dist/prettier-${version}-${require('./package.json').dependencies.prettier}.min.js`,
   format: 'cjs',
+  banner: version === 'cli' ? '#!/usr/bin/env node\n' : undefined,
   plugins: [
+    replace({
+      '#!/usr/bin/env node': '',
```

Comment:
You *should* be right but apparently it makes no difference in the output as the extra newline is already removed. Rollup must have some logic around this. I'm putting in the \n anyway though.